cql3/statements/create_table_statement.cc

```diff
@@ -452,6 +453,20 @@ std::optional<sstring> check_restricted_table_properties(
             break;
         }
     }
+    if (strategy && *strategy == sstables::compaction_strategy_type::time_window){
```

Comment:
Our coding style wants a space between `)` and `{`.

This should be in the strategy itself, though I see you followed existing practice here.